internal/runtime/storeforward.go

```diff
@@ -55,8 +57,12 @@ func (sf *storeForwardInfo) startStoreAndForwardRetryLoop(
 		defer appWg.Done()
 		defer enabledWg.Done()
 
-		retryInterval := config.Writable.StoreAndForward.RetryInterval
-		if retryInterval < defaultMinRetryInterval {
+		retryInterval, err := time.ParseDuration(config.Writable.StoreAndForward.RetryInterval)
+		if err != nil {
+			edgeXClients.LoggingClient.Warn(
+				fmt.Sprintf("StoreAndForward RetryInterval failed to parse, defaulting to %d", defaultMinRetryInterval))
+			retryInterval = defaultMinRetryInterval
+		} else if retryInterval < defaultMinRetryInterval {
 			retryInterval = defaultMinRetryInterval
```

Comment:
Need to also log warning here that interval too small.

Reply:
ok